src/blocks/modal.css

```diff
@@ -32,6 +32,38 @@
   /* margin: auto; */
   border-radius: 10px;
 }
+.modal__container-delete {
+  color: black;
+  background-color: #fff;
+  position: relative;
+  width: 430px;
+  min-height: 181px;
+  padding: 36px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 10px;
+  box-sizing: border-box; /*esto incluye padding y margin en el tamano total del elemento*/
+}
+.modal__container-picture {
+  color: black;
+  background-color: #fff;
+  position: relative;
+  width: 394px;
+  min-height: 236px;
+  padding: 36px;
+
+  /* margin: auto; */
+  border-radius: 10px;
+}
+.modal__button-delete {
```

Comment:
Needs correction: The button should also have the `pointer` cursor and the `hover` state